docs/content/production.html.md

```diff
@@ -273,6 +273,13 @@ from the exported app. If you'd like to provide a new name for the app, the
 app name and the cluster domain. To import the old routes in addition to the new
 route, add the `--routes` flag.
 
+### Recovery
+
+In the event that a cluster is having difficulties with apps, either internal
+or external, not starting after a cluster reboots, it may be necessary to run 
+the automated cluster repair tool. Run `flynn-host fix` from any node in the cluster
```

Comment:
Before running `flynn-host fix`, we should advise users to try and create a backup first if possible, and also post the gist created by `sudo flynn-host collect-debug-info` in a GitHub issue (or send to logs@flynn.io) so we can diagnose what caused the cluster to enter the failing state.